src/alice-renderer.d.ts

```diff
@@ -0,0 +1,78 @@
+declare module 'alice-renderer' {
```

Comment:
Declared Response interface is not-compatible with ApiResponseBody as a result tests won't work with the new reply.

I've tried to fix this issue in [forked repo](https://github.com/fletcherist/yandex-dialogs-sdk/compare/v3...shining-mind:v3), by declaring API as ambient module and importing API types here, but I can't find a good solution how to ship this types with NPM package. It seems that *.d.ts files added to dist folder won't be resolved by the typescript compiler if package is used as dependency.